archetypes/pattern/getting-started.adoc

```diff
@@ -0,0 +1,15 @@
+---
+title: Getting started
+weight: 10
+---
+
+:toc:
+:imagesdir: /images
+:_content-type: ASSEMBLY
+
+// Add asciidoc modules here
```

Comment:
Can we add a suggestion about using the [include::](https://docs.asciidoctor.org/asciidoc/latest/directives/include/) directive to add modules?